x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_panel.tsx

```diff
@@ -153,12 +156,22 @@ const SolutionNavPanelCategories: React.FC<SolutionNavPanelCategoriesProps> = ({
   );
 };
 
+const FlexLink = styled.a`
+  display: flex;
+  align-items: center;
+`;
+
+const BetaBadge = styled(EuiBetaBadge)<{ label: string; size?: 's' | 'm' }>`
+  margin-left: ${({ theme }) => theme.eui.euiSizeS};
+  color: ${(props) => props.theme.eui.euiTextColor};
+`;
```

Comment:
Would it be possible to move these components to _./x-solution_grouped_nav_panel.styles.tsx_, where styled components are defined?